shanoir-ng-anonymization/src/main/java/org/shanoir/anonymization/anonymization/AnonymizationServiceImpl.java

```diff
@@ -523,4 +484,25 @@ else if (vr.equals(VR.AE) || vr.equals(VR.AS) || vr.equals(VR.CS) || vr.equals(V
 		// VR.OD = Other Double String
 	}
 
+	private void anonymizeUID(int tagInt, Attributes attributes, Map<String, String> UIDs) {
```

Comment:
@LaurentPV the problem now is with the method factorization, that it produces too much logs
the info that the StudyInstanceUID has been generated, one by exam, is important, but one log
per serie is just too much, probably the reason, why I kept the 3 methods in the first place

Reply:
I added a condition to log only when studyInstanceUID is concerned. If you think that's really better with two different methods I'll rollback.